Gopkg.toml

```diff
@@ -34,20 +34,20 @@
   branch = "master"
 
 [[constraint]]
-  branch = "release-1.9"
   name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.11.2"
```

Comment:
Why did you go with the specific tag vs `release-1.11`? At least this will require searching for updates and improvements and the changes in `1.11.x` should be compatible.

Reply:
I considered this, but I always feel a little uneasy vendoring branches instead of specific tags. Eliminates the chance of upgrading unintentionally.